core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -204,32 +202,27 @@ var TopicEditorPage = function() {
   };
 
   this.changeSubtopicTitle = async function(title) {
-    await subtopicTitleField.clear();
+    await action.clear('Subtopic Title Field', subtopicTitleField);
     await subtopicTitleField.sendKeys(title);
   };
 
   this.changeSubtopicPageContents = async function(content) {
     await general.scrollToTop();
     var subtopicPageContentButton = element(by.css(
       '.protractor-test-edit-html-content'));
-    await waitFor.elementToBeClickable(
-      subtopicPageContentButton,
-      'Edit subtopic htm content button taking too long to be clickable');
-    await subtopicPageContentButton.click();
+    await action.click(
+      'Subtopic Page Content Button', subtopicPageContentButton);
     var pageEditor = element(by.css(
       '.protractor-test-edit-subtopic-page-contents'));
     await waitFor.visibilityOf(
       pageEditor, 'Subtopic html editor takes too long to appear');
     var pageEditorInput = pageEditor.element(by.css('.oppia-rte'));
-    await pageEditorInput.click();
-    await pageEditorInput.clear();
+    await action.click('Page Editor Input', pageEditorInput);
+    await action.clear('Page Editor Input', pageEditorInput);
     await pageEditorInput.sendKeys(content);
     var saveSubtopicPageContentButton = element(by.css(
       '.protractor-test-save-subtopic-content-button'));
-    await waitFor.elementToBeClickable(
-      saveSubtopicPageContentButton,
-      'Save Subtopic Content button taking too long to be clickable');
-    await saveSubtopicPageContentButton.click();
+    await action.click('Save Subtopic Button', saveSubtopicPageContentButton);
   };
 
   this.expectNumberOfUncategorizedSkillsToBe = async function(count) {
```

Comment:
Can we use `waitFor` statements here? Again, we only want to wait when `count` is nonzero